internal/proofs/convert.go

```diff
@@ -0,0 +1,98 @@
+package proofs
+
+import (
+	"fmt"
+	"math/bits"
+
+	ics23 "github.com/confio/ics23/go"
+	"github.com/tendermint/tendermint/crypto/merkle"
```

Comment:
Huh? Shouldn't this refer to the `MerkleMap` type you defined above?

Reply:
☝️

Reply:
this is referring `merkle.SimpleProof` from tendermint.